src/Microsoft.AspNet.Mvc.ViewFeatures/ValidateAntiForgeryTokenAttribute.cs

```diff
@@ -9,6 +9,14 @@
 
 namespace Microsoft.AspNet.Mvc
 {
+    /// <summary>
+    /// Specifies that the class or method that this attribute is applied validates the anti-forgery token.
```

Comment:
I prefer the original phrasing. The goal is to provide the information that helps the reader get their task completed. That is, "apply this attribute and anti-forgery will be validated." Nothing in this comment is incorrect.